test/aircraft/Pilot/Pilot.spec.js

```diff
@@ -619,14 +638,97 @@ ava('.conductInstrumentApproach() returns a success message', (t) => {
     t.deepEqual(result, expectedResult);
 });
 
+ava('.crossFix() returns early when the specified fix does not exist', (t) => {
+    const aircraftModel = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK);
+    const invalidFixNameMock = 'threeve';
+    const altitudeMock = 13000;
+    const expectedResult = [false, 'unable to find \'threeve\''];
+    const result = aircraftModel.pilot.crossFix(aircraftModel, invalidFixNameMock, altitudeMock);
+
+    t.deepEqual(result, expectedResult);
+});
+
+ava('.crossFix() returns early when the specified fix exists but is not on the aircraft\'s route', (t) => {
+    const aircraftModel = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK);
+    const nonRouteFixNameMock = 'ogkij';
+    const altitudeMock = 13000;
+    const expectedResult = [false, 'unable, \'ogkij\' is not on our route'];
+    const result = aircraftModel.pilot.crossFix(aircraftModel, nonRouteFixNameMock, altitudeMock);
+
+    t.deepEqual(result, expectedResult);
+});
+
+ava('.crossFix() returns early when the specified altitude fails .validateNextAltitude()', (t) => {
+    const aircraftModel = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK);
+    const fixNameMock = 'kepec';
+    const altitudeMock = 13000;
+    const expectedResult = [false, 'some reply from validate-next-altitude'];
+
+    sinon.stub(aircraftModel, 'validateNextAltitude', () => expectedResult);
```

Comment:
could we use the `returns` syntax here instead? 
```suggestion
    sinon.stub(aircraftModel, 'validateNextAltitude').returns(expectedResult);
```

I find, as usual, this is a little easier to read

Reply:
works for me! done